data/demoData.csv

```diff
@@ -0,0 +1,105 @@
+,farmerName,treeType,numCuerdas,ageOfTrees
```

Comment:
leave this as is for now, but it occurs to me now that `ageofTrees` might as well just be `yearPlanted`